src/sdltiles.cpp

```diff
@@ -24,8 +24,10 @@
 
 #if defined(_MSC_VER) && defined(USE_VCPKG)
 #   include <SDL2/SDL_image.h>
+#   include <SDL2/SDL_syswm.h>
 #else
 #   include <SDL_image.h>
+#   include <SDL_syswm.h>
```

Comment:
This include messes with Linux builds -- it pulls in X11's `X.h`, which causes a multiple definition error of `class Font`.

Reply:
```
src/sdltiles.cpp:104:7: error: definition of type 'Font' conflicts with typedef of the same name
class Font
      ^
/usr/include/X11/X.h:100:13: note: 'Font' declared here
typedef XID Font;
            ^
```

Reply:
Thanks, fixed!